src/components/dialogs/pick-entity-or-conversion.vue

```diff
@@ -81,13 +81,36 @@
 </template>
 
 <script lang="ts">
-import newReqModule from '@/store/new-request-module'
 import { Component, Vue, Watch } from 'vue-property-decorator'
-import { SelectOptionsI } from '@/interfaces'
+import { Action, Getter } from 'vuex-class'
+
+import { ConversionTypesI, EntityI, SelectOptionsI } from '@/interfaces'
+import { ActionBindingIF } from '@/interfaces/store-interfaces'
+import { ConversionTypes } from '@/store/list-data'
 
 @Component({})
 export default class PickEntityOrConversionDialog extends Vue {
-  showSocietiesInfo = false
+  // Global Getters
+  @Getter getConversionTypeOptions!: ConversionTypesI[]
+  @Getter getEntityBlurbs!: Array<EntityI>
+  @Getter getEntityTypeCd!: string
+  @Getter getEntityTypeOptions!: Array<EntityI>
+  @Getter getEntityTypesBC!: Array<EntityI>
+  @Getter getEntityTypesXPRO!: Array<EntityI>
+  @Getter getIsConversion!: boolean
+  @Getter getLocation!: string
+  @Getter getLocationText!: string
+  @Getter getPickEntityModalVisible!: boolean
+
+  // Global Actions
+  @Action setConversionType!: ActionBindingIF
+  @Action setConversionTypeAddToSelect!: ActionBindingIF
+  @Action setEntityTypeCd!: ActionBindingIF
+  @Action setEntityTypeAddToSelect!: ActionBindingIF
+  @Action setPickEntityModalVisible!: ActionBindingIF
+
+  private showSocietiesInfo = false
+  private conversionTypes = ConversionTypes
```

Comment:
Haha, so much better than
```
get conversionTypes () {
    return newReqModule.conversionTypes
  }
```

If this is just to make ConversionTypes visible to the template, you can use:
`readonly ConversionTypes = ConversionTypes`